<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>request_form</class>
 <widget class="QDialog" name="request_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>460</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>749</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::FieldsStayAtSizeHint</enum>
    </property>
    <property name="rowWrapPolicy">
     <enum>QFormLayout::DontWrapRows</enum>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="cerroLabel">
      <property name="text">
       <string>Cerro de destino</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="cerroComboBox">
      <item>
       <property name="text">
        <string>-- seleccione cerro --</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Alegre</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Barón</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Blanco</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bellavista</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Concepción</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Delicias</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>El Litre</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>El Molino</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Esperanza</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Jiménez</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Larraín</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>La Cruz</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>La Cárcel</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>La Florida</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>La Merced</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>La Virgen</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Las Cañas</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Las Jarcias</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Las Monjas</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Los Placeres</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Loceras</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lecheros</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mariposas</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mesilla</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Miraflores</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>O'Higgins</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pajonal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Panteón</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Playa Ancha</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Perdices</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Polanco</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ramaditas</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Reina Victoria</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rodelillo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rocuant</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San Juan de Dios</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San Francisco</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Toro</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Yungay</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="payLabel">
      <property name="text">
       <string>Medio de Pago</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="payComboBox">
      <item>
       <property name="text">
        <string>-- seleccione medio de pago --</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Efectivo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cheque</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RedCompra</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Medios de pago no tradicionales</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Cilindros</string>
      </property>
      <widget class="QCommandLinkButton" name="commandLinkButton">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>0</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Ingresar</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_kg">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>0</y>
         <width>85</width>
         <height>27</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>5 Kg</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11 Kg</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15 Kg</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>45 Kg</string>
        </property>
       </item>
      </widget>
      <widget class="QSpinBox" name="spinBox_quantity">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>0</y>
         <width>48</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>0</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Cantidad</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>0</y>
         <width>71</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Capacidad</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_catalitic">
       <property name="geometry">
        <rect>
         <x>424</x>
         <y>0</y>
         <width>151</width>
         <height>27</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Catalitico</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>No Catalitico</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Vista Previa Pedido</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPlainTextEdit" name="previewText">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>601</width>
     <height>291</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>492</width>
     <height>192</height>
    </size>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>----------    FLIPIGAS LTDA.    ----------

Solicitud n°:	%

Cerro de Destino:	%

Medio de Pago:	%

	Detalle: 

Tiempo estimado de entrega: %

TOTAL: %
</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>request_form</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>request_form</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
